comics/comics/goblins.py

```diff
@@ -10,14 +10,9 @@ class ComicData(ComicDataBase):
     rights = "Tarol Hunt"
 
 
-class Crawler(CrawlerBase):
+class Crawler(ComicControlCrawlerBase):
     history_capable_days = 30
     time_zone = "US/Pacific"
 
     def crawl(self, pub_date):
-        feed = self.parse_feed("http://www.goblinscomic.com/feed/")
-        for entry in feed.for_date(pub_date):
-            if "Comics" not in entry.tags:
-                continue
-            url = entry.summary.src('img[src*="/comics/"]')
-            return CrawlerImage(url)
+        return self.crawl_helper("http://www.goblinscomic.com", pub_date)
```

Comment:
I guess most of these could use `ComicData.url` instead of repeating the URL here.